### PR TITLE
Creating iconButton and his functionality

### PR DESCRIPTION
**Pull request by santiago-jv**
Please check this branch for it will merge.

### CODE REVIEW

File: public/views/home_page.html [hunk 1]
```diff
@@ -55,7 +55,13 @@ <h1 class="logo">
           
         </article>
       </main>
+      <div class="iconButtonContainer">
+        <i onclick="slideDown()" class="fas fa-arrow-circle-down"></i>     
+     </div>
      
+    </section>
+    <section id="another-section">
```
Comment (LoreA-dev) [inline public/views/home_page.html]: Use properly names for the tags, please. 

In this case, i don't know if is correct to call a section "another-section".

Comment (santiago-jv) [reply public/views/home_page.html]: It's just a sample but I already remove it

Comment (santiago-jv) [reply public/views/home_page.html]: Fixed inconvenience. Please merge this branch.

**Review by LoreA-dev — APPROVED** [review]
Nice 🤘🤘

**Pull request merged**